main.tex

```diff
@@ -432,6 +436,54 @@ \section{Using Domain-Specific Languages to improve the
 without building an expensive prototype, and advanced types of hosting language
 encoding initial system specification provide verified-by-construction result. 
 
+\section{Tool support}
+
+Formalising the design specifications is a good approach to the development of a resilient
+system. Specifications, represented via mathematical models, can indeed be verified and
+simulated, targeting an error-free system. Workcraft provides the support for different
+models, this can help the design for resiliency. Conditional partial order graphs (CPOG)
+(available in the Workcraft framework), for instance, can support the development of
+processor instruction sets. In Figure \ref{fig:screenshot}, all the steps from the
+specification of each instruction, to the synthesis of the final microcontroller are
+depicted. Instructions can be specified in the form of algebraic equations at first, then the
+graphs can be imported, encoded and synthesised in the form of a microcontroller. The latter
+can be mapped with the usage of a technology library and eventually simulated: internal and
+external signals can be visualised via digital waveforms, where dependencies among signals
+are shown. The development of a domain specific language (DSL), that can be used for the
+specification of the instructions, still remains a gap of this flow. Having a scalable, yet
```

Comment:
> still remains a gap of this flow

I would rephrase this in a more positive light: we are developing a DSL for bridging the gap between formal methods toolsets such Rodin and hardware synthesis toolsets such as Workcraft.